city_scrapers/spiders/il_police_professionalism.py

```diff
@@ -20,46 +20,70 @@ def parse(self, response):
         Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
         needs.
         """
-        for item in response.xpath("//*[contains(text(),\"Police Professionalism\")]"):
-            meeting = Meeting(
-                title=self._parse_title(item),
-                description=self._parse_description(item),
-                classification=self._parse_classification(item),
-                start=self._parse_start(item),
-                end=None,
-                all_day=self._parse_all_day(item),
-                time_notes=self._parse_time_notes(item),
-                location=self._parse_location(item),
-                links=self._parse_links(item),
-                source=self._parse_source(response),
-            )
-
-            meeting["status"] = self._get_status(meeting)
-            meeting["id"] = self._get_id(meeting)
-
-            yield meeting
+        list_items = response.xpath("//li[contains(@style,'padding-bottom')]")
+        relavent_links = list_items.xpath(
+            'a[(.//*|.)[contains(text(),"Police Professionalism")]]/@href'
+        ).extract()
+        # titles = relavent_links = list_items.xpath(
+        #     'a[(.//*|.)[contains(text(),"Police Professionalism")]]/text'
+        # ).extract()
+        for i in range(0, len(relavent_links)):
+            link = response.urljoin(relavent_links[i])
+            yield response.follow(link, self._parse_item)
+
+    def _parse_item(self, response):
+        """`_parse_item` should always `yield` Meeting items"""
+        paragraphs = response.xpath('//p')
+        title = response.xpath('//h3/text()').get()
+        meeting = Meeting(
+            title=self._parse_title(response),
+            description=self._parse_description(response, paragraphs),
+            classification=self._parse_classification(title),
+            start=self._parse_start(response, paragraphs),
+            end=None,
+            time_notes='',
+            all_day=False,
+            location=self._parse_location(response),
+            links=self._parse_links(response),
+            source=response.url,
+        )
+        meeting['id'] = self._get_id(meeting)
+        meeting['status'] = self._get_status(meeting, title)
+        return meeting
 
     def _parse_title(self, item):
         """Parse or generate meeting title."""
         return "Commission on Police Professionalism"
 
-    def _parse_description(self, item):
+    def _parse_description(self, item, paragraphs):
+        return_list = []
+        # paragraphs = paragraphs[0:len(paragraphs) - 1]
+        for i in paragraphs:
+            xpath_selector = i.xpath('text()').get()
+            if xpath_selector is not None:
+                return_list.append(xpath_selector.strip().replace(u'\xa0', u''))
+        final_description = " ".join(return_list)
```

Comment:
Small thing, but you can just return this directly instead of assigning it to a variable first